language-server/modules/langserver-core/src/test/resources/completion/variable-declaration/config/var_def_with_anon_func_config4.json

```diff
@@ -585,7 +585,7 @@
     {
       "label": "testFunc",
       "kind": "Variable",
-      "detail": "function (int i) returns function (int i, string s) returns int",
+      "detail": "function ($CompilationError$ i) returns function (int i, $CompilationError$ s) returns int",
```

Comment:
We shouldn't get the `$CompilationError$` as the type

Reply:
With new changes, the compiler gives `redeclared symbol ` error for the `s` and `i` parameters. Here, isn't it the reason to give `CompilationError`?

Reply:
@chiranSachintha yes, in that case shall we change the particular test source `var_def_with_anon_func_source4.bal`? So the test will reflect the proper values.